app/styles/style.css

```diff
@@ -54,25 +59,470 @@ body {
+  padding: 8px;
+  width: 18px;
+  margin-top: 18px;
+}
+
+.about-resume {
+  padding-top: 56px;
+}
+
+.about-categories {
+  display: flex;
+  flex-direction: column;
+  margin-top: 36px;
+  width: 50%;
+  padding: 20px;
+  padding-left: 0;
+}
+
+.section-title {
+  width: 100%;
+  height: 72px;
+  display: flex;
+  padding: 10px;
+  align-items: center;
+  justify-content: space-between;
+  font-family: Poppins;
+}
+
+.title-text {
+  font-size: 20px;
+}
+
+.dropdown-icon {
+  width: 50px;
+}
+
+.categories {
+  margin: 0;
+  padding: 10px;
+  display: flex;
+}
+
+.category-item {
+  background-color: #f4f5f7;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin: 10px;
+  margin-bottom: 12px;
+  padding: 10px;
+  width: 122px;
+  height: 120px;
+}
+
+.category-img .cat-img {
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
+  padding: 10px;
+}
+
+.cat-name {
+  margin-left: 10px;
+  font-size: 15px;
+  font-family: PoppinsMedium;
+  font-weight: 400;
+  color: #253858;
+  padding-top: 10px;
+}
+
+.about-body {
+  font-size: 16px;
+  font-family: Poppins;
+  line-height: 24px;
+  font-weight: 400;
+  color: #344563;
+  padding-right: 24px;
+}
+
+.divide {
+  border: 1px #dfe1e6 solid;
+}
+
+/* contact section design starts here */
+.contact-section {
+  background-color: #fff;
+  width: 100%;
+}
+
+.contact-container {
+  background-image: url("./../assets/pngs/icons/contactBgDesktop.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #6070ff;
+  border-radius: 94px 0 0 0;
+  display: flex;
+  justify-content: center;
+  margin-top: 88px;
+  width: 100%;
+}
+
+.contact-contents {
+  margin-top: 114px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: 50%;
+}
+
+.contact-title h2 {
+  margin: 0;
+  margin-top: 114px;
+  color: #fff;
+  font-family: PoppinsBold;
+  font-size: 40px;
+}
+
+.contact-body {
+  width: 70%;
+}
+
+.contact-body p {
+  color: #fff;
+  font-size: 20px;
+  font-family: Poppins;
+  line-height: 28px;
+}
+
+.form-container {
+  width: 80%;
+}
+
+.input-container {
+  padding: 10px;
+}
+
+.input-field {
+  width: 100%;
+  border-radius: 8px;
+  border: none;
+  height: 48px;
+  padding-left: 20px;
+  box-sizing: border-box;
+}
+
+.input-field:focus {
+  border-radius: none;
+  border: 3px red solid;
+}
+
+.text-field {
+  border-radius: 8px;
+  border: 0 solid;
+  height: 170px;
+  width: 100%;
+  padding-left: 20px;
+  padding-top: 20px;
+}
+
+.button-container {
+  padding-left: 8px;
+  padding-bottom: 70px;
+  display: flex;
   justify-content: flex-start;
-  height: 604px;
+  align-items: center;
 }
 
-@media (max-width: 540px) {
-  .main-header {
-    width: 100%;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    align-content: center;
-    height: 71px;
-  }
+.contact-btn {
+  background-color: #fff;
+}
 
+@media (max-width: 720px) {
```

Comment:
- [ ] Your design on mobile is not consistent from the header to the footer as provided by the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=tAY7arxQ0PcdihlD-0) mainly because it is still using the styling you used for the desktop to style the mobile, In your project requirement as shown below the media queries that you are required to use to affect the way your website would look on a desktop version is `768px` but from the highlighted line you are making use of `720px` please kindly fix this issue.
- [ ] With respect to the use of media queries it is expected of you to make use of `min-width: 768px` to style anything in the desktop view rather than making use of `max-width` because by using max-width whatever styling placed here would affect any device below the specified like what is currently happening with your deign please fix this design flaw issue before resubmitting your next review request. :+1: :sparkles: 
![req](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/89055502/611c0d29-5e53-48c1-80db-936653283b04)
- figma design
![f-q](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/89055502/8caaf60e-9319-4862-a152-f9afc0a97b01)

- your mobile designs 
![s-1](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/89055502/77207b46-440a-4619-a24c-c7b400f3d073)
![s-2](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/89055502/3a7cd026-9d32-4ba9-9085-9990b2739878)
![s-3](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/89055502/f92f2b0b-c70a-439d-824b-241529c2ab0e)